api/contracts.go

```diff
@@ -121,23 +128,26 @@ func Native(ctx context.Context, client *ethclient.Client, root *bind.TransactOp
 	if err = client.SendTransaction(ctx, tx); err != nil {
 		return err
 	}
-	return storeBlockResultsForTxs(ctx, client, "native_transfer", tx)
+	return storeBlockResultsForTxs(ctx, client, TRACEDATA_DIR_PREFIX+"native/", "native_transfer", tx)
 }
 
 func NewERC20(ctx context.Context, client *ethclient.Client, root, auth *bind.TransactOpts) error {
 	_, tx, erc20Token, err := erc20.DeployERC20Template(root, client, root.From, root.From, "WETH coin", "WETH", 18)
 	if err != nil {
 		return err
 	}
-	if err = storeBlockResultsForTxs(ctx, client, "erc20_deploy", tx); err != nil {
+
+	path := TRACEDATA_DIR_PREFIX + "erc20/"
+	if err = storeBlockResultsForTxs(ctx, client, path, "erc20_deploy", tx); err != nil {
```

Comment:
```suggestion
	if err = storeBlockResultsForTxs(ctx, client, path, "deploy", tx); err != nil {
```
we can remove the json name prefix now